headers/types/files/wan.h

```diff
@@ -95,4 +95,34 @@ struct wan_header {
 #pragma pack(pop)
 ASSERT_SIZE(struct wan_header, 10);
 
+struct wan_fragment {
+    int16_t fragment_bytes_index; // negative mean using the previous defined fragment bytes (or to
+                                  // not update it?)
```

Comment:
Let's move this comment above the field so we don't need to break the comment

Reply:
Done